tests/lifecycle/tests/lifecycle_shutdown.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/networking/nethelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/deployment"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("lifecycle lifecycle-container-shutdown", func() {
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(lifeparameters.LifecycleNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 47311
+	It("One deployment, one pod with one container that has preStop field configured", func() {
+
+		By("Define deployment with preStop field configured")
+		preStopCommand := []string{"/bin/sh", "-c", "killall -0 tail"}
```

Comment:
We use this var only in 1 place. Please remove this var and paste this command directly into func's argument field